app/scripts/components/modeldetails.js

```diff
@@ -112,6 +113,9 @@ var exports = (function () {
         e.stopPropagation();
         $(e.target).closest(".panel").children(".collapse").collapse("toggle");
       },
+      getActiveModelData: function (str) {
```

Comment:
Hmmm, not really: I'd like to keep all javascript in the .js file and work with human readable identifiers (properties, computed properties and methods) in the template. 

I like your suggestion though. Never would've guessed it would be that easy and the solution is indeed elegant.